grandpapa_board.X/main.c

```diff
@@ -193,6 +201,23 @@ static void send_status_ok_mcp(void) {
     mcp_can_send(&board_stat_msg);
 }
 
+static void send_log_msgs(void) {
+    can_msg_t batt_cur_msg;
+    uint16_t batt_cur = ADCC_GetSingleConversion(channel_BATT_CURR)/2;
```

Comment:
Magic numbers should not be living in the code like this. When I read this code, I know that you're dividing by 2 based on component values on the board, but anyone else probably doesn't. Consider putting this into a macro (`#define CURR_AMP_FACTOR 0.5`) and perhaps a comment indicating where the value comes from